src/test/VesperAdapterV1.t.sol

```diff
@@ -308,13 +336,8 @@ contract VesperAdapterV1Test is DSTestPlus {
             amountDAI < type(uint96).max
         );
 
-        hevm.assume(
-            amountUSDC >= 10**SafeERC20.expectDecimals(address(weth)) && 
-            amountUSDC < type(uint96).max
-        );
-
         deal(address(DAI), address(this), amountDAI);
-        deal(address(USDC), address(this), amountUSDC);
+        deal(address(USDC), address(this), amountDAI);
```

Comment:
why the switch here? this would give us the wrong number of decimal places for usdc

Reply:
When I pushed to github it was saying that assume was overloaded so I was trying to reuse data but overlooked the decimals